xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -348,3 +371,38 @@ void CAndroidUtils::OnSettingChanged(const std::shared_ptr<const CSetting>& sett
   if (settingId == CAndroidUtils::SETTING_LIMITGUI)
     CDisplaySettings::GetInstance().ClearCalibrations();
 }
+
+const std::vector<int> CAndroidUtils::GetDisplaySupportedHdrTypes()
+{
+  CJNIWindow window = CXBMCApp::getWindow();
+
+  if (window)
+  {
+    CJNIView view = window.getDecorView();
+    if (view)
+    {
+      CJNIDisplay display = view.getDisplay();
+      if (display)
+      {
+        CJNIDisplayHdrCapabilities caps = display.getHdrCapabilities();
+        if (caps)
+          return caps.getSupportedHdrTypes();
+      }
+    }
+  }
+
+  return {};
+}
+
+const void CAndroidUtils::LogDisplaySupportedHdrTypes()
```

Comment:
just `void` please

Reply:
same:
`void CAndroidUtils::LogDisplaySupportedHdrTypes() const`

Reply:
what thexai said was the intention of my comment "Could this member function be const?".